client/stylesheets/_toCat.scss

```diff
@@ -3499,8 +3586,95 @@ On click the images will be displayed at normal size to complete the effect
     margin: 0px 0 0 10px;
     vertical-align: middle;
     color: #fff !important;
+} */
+
+.profileChatBtn a {
+    font-size: 1.2rem;
+    padding: 15px 22px;
+    color: #fff !important;
+    background-color: #f8504b;
+    border: 1px solid #f8504b;
```

Comment:
Color literals like `#f8504b` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces